app/js/config-service.js

```diff
@@ -5,7 +5,7 @@ angular.module('cttvServices').
 
 
 
-    /**
+    /**º
```

Comment:
I think the extra char might have been introduced by mistake? although since it's inside the comment block it shouldn't break anything